tests/components/advantage_air/test_sensor.py

```diff
@@ -123,7 +123,7 @@ async def test_sensor_platform(
         dt_util.utcnow() + timedelta(seconds=RELOAD_AFTER_UPDATE_DELAY + 1),
     )
     await hass.async_block_till_done()
-    assert len(mock_get.mock_calls) == 2
+    assert len(mock_get.mock_calls) == 3
```

Comment:
Why would this change?

Reply:
To be honest, I dont know. I'll try figure out why its getting called 3 times in sensor but not binary sensor.

Reply:
Ok I can explain this now.

Previously when there was no debounce, a refresh would have been done immediately before the attempt to enable the disabled entitiy, but with the debounce it fell in the RELOAD_AFTER_UPDATE_DELAY window.

I have broken this test out, and now its back to only triggering twice, which is expected.